packages/common-utils/src/renderChartConfig.ts

```diff
@@ -746,7 +745,12 @@ function renderWith(
   if (withClauses) {
     return concatChSql(
       ',',
-      withClauses.map(clause => chSql`${clause.name} AS (${clause.sql})`),
+      withClauses.map(clause => {
+        if (clause.isSubquery) {
+          return chSql`${{ Identifier: clause.name }} AS (${clause.sql})`;
+        }
+        return chSql`(${clause.sql}) AS ${{ Identifier: clause.name }}`;
```

Comment:
using with variable to bypass alias is quite clever! 🤯

Reply:
oops I inverted the if conditions for default value 🤦